util/util.go

```diff
@@ -110,6 +121,131 @@ func JSONOneLine(obj interface{}) string {
 	return string(b)
 }
 
```

Comment:
Where did the code in these functions come from? Not being covered by tests is a little concerning.

Reply:
these are from go-algorand. I can add tests for these.

Reply:
tests added